packages/theme/template/Icon/Icon.js

```diff
@@ -49,7 +49,16 @@ const IconBase = kind({
 		 * @type {String|Object}
 		 * @public
 		 */
-		children: PropTypes.oneOfType([PropTypes.string, PropTypes.object])
+		children: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal elements and states of this component.
+		 *
+		 * @type {Object}
```

Comment:
Why is this private? Would you please list the supported public class names here?

Reply:
I pushed a change, but I am a bit confused about the supported public classnames. If `publicClassnames` is not mentioned in the styles block, are there any classnames which are public?

Reply:
Hm.. Did you put this change to Icon the first time because of the linting warning?
After seeing your comment, I think `css={props.css}` is here to override ui/Icon's style not to make this component overridable via public classes. It's the same for Heading and Checkbox.
WDYT?

Reply:
I only added the propType only because of the lint warning that was thrown before:
64:15  warning  'css' is missing in props validation  enact/prop-types

Reply:
what do you think is the best solution in this case?